cmake/modules/hunter_config.cmake

```diff
@@ -8,6 +8,61 @@ include(hunter_pack_git_submodule)
 include(hunter_unsetvar)
 include(hunter_user_error)
 
+function(parse_cmake_args_for_keyword cmake_args)
```

Comment:
Currently rule is one module - one function. Name of the function should match module and start with `hunter_` to avoid clash with user space or standard CMake modules.